### PR TITLE
Update manpages for 1.14.8 release

### PR DESCRIPTION
**Pull request by chromatic**

### CODE REVIEW

**Review by edtubbs — APPROVED** [review]
ACK, changes look good.. dogecoin-qt runs on 32-bit windows, but since man pages are for linux this change makes sense.
https://github.com/dogecoin/dogecoin/blob/0a3ba38f3813df2d66d5f9fe72fc90860c64e569/doc/man/dogecoin-qt.1#L6

File: doc/man/dogecoin-qt.1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
-.TH DOGECOIN-QT "1" "February 2024" "dogecoin-qt v1.14.7.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.3.
+.TH DOGECOIN-QT "1" "July 2024" "dogecoin-qt v1.14.8.0" "User Commands"
 .SH NAME
-dogecoin-qt \- manual page for dogecoin-qt v1.14.7.0
+dogecoin-qt \- manual page for dogecoin-qt v1.14.8.0
 .SH DESCRIPTION
-Dogecoin Core version v1.14.7.0
+Dogecoin Core version v1.14.8.0 (64\-bit)
```
Comment (patricklodder) [inline doc/man/dogecoin-qt.1:6]: Let's remove the 64-bit. The manage is also valid for 32-bit.

**Comment by patricklodder** [comment]
This is good to go after squash

**Comment by chromatic** [comment]
squashed

**Review by edtubbs — APPROVED** [review]

**Review by patricklodder — APPROVED** [review]

**Comment by patricklodder** [comment]
utACK.

**Pull request merged**